src/crossroads/main.py

```diff
@@ -184,9 +194,97 @@ def validate_json(json_content):
         return False
     return True
 
+def lists_have_common_parts(first_data_list, second_data_list):
+    """
+    Checks that the dates in both dictionary lists have are common parts.
+
+    :param first_data_list: (list) first user dictionaries list with coordinates
+    :param second_data_list: (list) second user dictionaries list with coordinates
+    :return: (bool) True or False
+    """
+
+    if int(first_data_list[-1]['timestamp']) < int(second_data_list[0]['timestamp']) or\
+            int(second_data_list[-1]['timestamp']) < int(first_data_list[0]['timestamp']):
+        return False
+    else:
+        return True
+
+def consolation_coordinates(first_data_list, second_data_list):
```

Comment:
Why you are extracting only dates from lists? Coordinates will be required in next tickets.